### ISSUE TITLE
ONNX export failed: Couldn't export Python operator Transfer

### ISSUE THREAD

**Issue by wangyunxiaa** [pytorch/pytorch#12529]
I am trying to export a pretrained pytorch model to onnx format. I got an error as following:
Graph we tried to export:
graph(%input_1 : Long(1, 7)
      %input_2 : Long(1)
      %learned_0 : Float(118427, 512)
      %learned_1 : Float(6144, 512)
      %learned_2 : Float(6144, 1536)
      %learned_3 : Float(6144)
      %learned_4 : Float(6144)
      %learned_5 : Float(512, 1536)
      %learned_6 : Float(118427)
      %learned_7 : Float(118427)) {
  %10 : Float(1, 7, 512) = onnx::Gather(%learned_0, %input_1), scope: RNNModel/Embedding[encoder]
  %11 : Float(1, 7, 512) = ^Transfer(None)(%10), scope: RNNModel
  %12 : Float(1, 7, 512) = onnx::Slice[axes=[0], ends=[9223372036854775807], starts=[0]](%11), scope: RNNModel
  %13 : Float(1!, 6, 512) = onnx::Slice[axes=[1], ends=[-1], starts=[0]](%12), scope: RNNModel
  %14 : Float(1!, 6, 512), %15 : Dynamic = onnx::Dropout[ratio=0](%13), scope: RNNModel/Dropout[drop]
  %16 : Long() = onnx::Constant[value={0}]()
  %17 : Dynamic = onnx::Shape(%14), scope: RNNModel/LSTM[rnn]
  %18 : Long() = onnx::Gather[axis=0](%17, %16), scope: RNNModel/LSTM[rnn]
  %19 : Long() = onnx::Constant[value={1}]()
  %20 : Long() = onnx::Constant[value={1536}]()
  %21 : Dynamic = onnx::Unsqueeze[axes=[0]](%19), scope: RNNModel/LSTM[rnn]
  %22 : Dynamic = onnx::Unsqueeze[axes=[0]](%18), scope: RNNModel/LSTM[rnn]
  %23 : Dynamic = onnx::Unsqueeze[axes=[0]](%20), scope: RNNModel/LSTM[rnn]
  %24 : int[] = onnx::Concat[axis=0](%21, %22, %23), scope: RNNModel/LSTM[rnn]
  %25 : Float(1, 1, 1536) = onnx::ConstantFill[dtype=1, input_as_shape=1, value=0](%24), scope: RNNModel/LSTM[rnn]
  %26 : Long() = onnx::Constant[value={1}]()
  %27 : Long() = onnx::Constant[value={1}]()
  %28 : Double() = onnx::Constant[value={0}]()
  %29 : Long() = onnx::Constant[value={0}]()
  %30 : Long() = onnx::Constant[value={0}]()
  %31 : Long() = onnx::Constant[value={1}]()
  %32 : Float(1!, 6, 1536), %33 : Float(1, 1, 1536), %34 : Float(1, 1, 1536) = aten::lstm(%14, %25, %25, %learned_1, %learned_2, %learned_3, %learned_4, %26, %27, %28, %29, %30, %31), scope: RNNModel/LSTM[rnn]
  %35 : Float(1!, 6, 1536), %36 : Dynamic = onnx::Dropout[ratio=0](%32), scope: RNNModel/Dropout[drop]
  %37 : Float(1536!, 512!) = onnx::Transpose[perm=[1, 0]](%learned_5), scope: RNNModel/Linear[projection]
  ......
  ......
  %55 : Dynamic = prim::Undefined(), scope: RNNModel/IndexLinear[criterion]/CrossEntropyLoss[ce]
  %56 : Long() = onnx::Constant[value={0}]()
  %57 : Long() = onnx::Constant[value={-100}]()
  %58 : Float(6) = aten::nll_loss(%54, %53, %55, %56, %57), scope: RNNModel/IndexLinear[criterion]/CrossEntropyLoss[ce]
  %output : Float(1, 6) = aten::view_as(%58, %40), scope: RNNModel/IndexLinear[criterion]
  return (%output);
}
the error of the op is  "%11 : Float(1, 7, 512) = ^Transfer(None)(%10), scope: RNNModel"
